src/composables/deepLinkApi.ts

```diff
@@ -26,8 +28,11 @@ export function useDeepLinkApi({ router }: UseDeepLinkApiOptions) {
   /**
    * Function needed to support legacy tipping from superhero.com
    */
-  function checkIfOpenTransferSendModal() {
-    if (checkIfSuperheroCallbackUrl(route.query)) {
+  async function checkIfOpenTransferSendModal(currentRoute: Route) {
+    if (
+      currentRoute.path.slice(1) === ROUTE_ACCOUNT
```

Comment:
Should we open the transfer modal only on the account page?

Reply:
This check is used only on the `/account` page. And the legacy code we are trying to support is aiming only for this page.